Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -371,6 +371,31 @@ public let CodeGenerators: [CodeGenerator] = [
         b.reassign(target, to: val)
     },
 
+    CodeGenerator("DestructArrayGenerator", input: .iterable) { b, arr in
+        // TODO: We need a way to expose the length of arr so that element selection is accurate
+        // Pick random indices
+        var indices: [Int] = []
+        for idx in 0..<Int.random(in: 0..<5) {
```

Comment:
Maybe add a short comment about how these random values are determined (5 and 0.7). I think 5 makes sense since it's the default max length of array literals (is it?). Then the probability should probably be selected so that on average, we define 2-3 new variables?

Reply:
The max length for array generation that we currently have is 10 elements, Should we continue with 5 as the upper limit for array destruct?

Reply:
Right ok, yeah maybe still leave it at 5 then, since that's the expected number of elements in an array if the max is 10. WDYT?

Reply:
Yeah that works for me too